client/lib/protect-form/index.tsx

```diff
@@ -1,25 +1,24 @@
 /**
  * External dependencies
  */
-import React, { Component, ComponentType } from 'react';
+import React, { ComponentType } from 'react';
 import debugModule from 'debug';
 import page from 'page';
 import i18n from 'i18n-calypso';
-import { includes, without } from 'lodash';
 import { Subtract } from 'utility-types';
 
 /**
  * Module variables
  */
 const debug = debugModule( 'calypso:protect-form' );
 
-type ComponentMarkedWithFormChanges = Component;
+type FormId = [  ];
```

Comment:
I see `wp-prettier` has a bug when formatting TypeScript's `TupleTypeAnnotation`. There should be no spaces inside `[]` 🙂